lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -211,12 +253,44 @@ class ClassToMixinConverter {
       if (shouldSwapParentClass) {
         yieldPatch(
             node.extendsClause.superclass.name.offset,
-            node.extendsClause.superclass.name.end,
+            node.extendsClause.superclass.end,
```

Comment:
This ensures that if the old superclass was parameterized, that those parameters don't end up on `UiProps` / `UiState`.